packages/integration-sdk-cli/src/generator/template/.github/workflows/build.yml.hbs

```diff
@@ -49,5 +49,5 @@ jobs:
       - name: Build and Release
         uses: jupiterone/action-npm-build-release@v1
         with:
-          npm_auth_token: ${{ secrets.NPM_AUTH_TOKEN }}
-          gh_token: ${{ secrets.AUTO_GITHUB_PAT_TOKEN }}
+          npm_auth_token: $\{{ secrets.NPM_AUTH_TOKEN }}
```

Comment:
The generator was generating these properties empty as `{{}}` is a handlebars expression

Reply:
Nice catch!